src/ucontext.rs

```diff
@@ -22,4 +23,13 @@ impl UContext {
         };
         Errno::result(res).map(drop)
     }
+
+    #[allow(mutable_transmutes)]
+    pub fn mut_sigmask(&mut self) -> &mut SigSet {
```

Comment:
Another example that supports my choice of name:

https://github.com/rust-lang/rust/commit/9370d3a05107bdcac6fb62d3bd5380e1d7c79b5b